components/model-info.tsx

```
@@ -370,22 +512,212 @@ export function ModelInfo() {
+                const layerNumbers = new Set();
+                for (const key in singleGroup.properties) {
+                  if (key.startsWith("Layer_")) {
+                    const parts = key.split("_");
+                    if (parts.length > 1) {
+                      layerNumbers.add(parts[1]); // Add the layer number string
+                    }
+                  }
+                }
+                if (
+                  layerNumbers.size <= 1 &&
+                  Object.keys(singleGroup.properties).hasOwnProperty(
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)